deploy/docker/base.dockerfile

```
@@ -60,6 +60,9 @@ RUN mv /opt/caddy/caddy /opt/caddy/caddy_vanilla
 
 COPY --from=caddybuilder /usr/bin/caddy /opt/caddy/caddy
 
+# Add newrelic related files
+RUN curl -o /opt/newrelic/newrelic.jar https://download.newrelic.com/newrelic/java-agent/newrelic-agent/8.9.1/newrelic-agent-8.9.1.jar
```

Can we ensure the parent folder exists, or are we doing that somewhere already? 🤔

```suggestion
RUN mkdir -p /opt/newrelic \
  curl -o /opt/newrelic/newrelic.jar https://download.newrelic.com/newrelic/java-agent/newrelic-agent/8.9.1/newrelic-agent-8.9.1.jar
```

---

Ah I see, yeah that fine then. Thanks.